public/app/features/stars/StarToolbarButton.tsx

```diff
@@ -1,109 +1,71 @@
 import { useMemo } from 'react';
-import { useAsyncRetry } from 'react-use';
 
 import { selectors } from '@grafana/e2e-selectors';
 import { t } from '@grafana/i18n';
-import { config, getBackendSrv } from '@grafana/runtime';
 import { Icon, ToolbarButton } from '@grafana/ui';
-import { useAddStarMutation, useRemoveStarMutation, useListStarsQuery } from 'app/api/clients/preferences/v1alpha1';
-import { contextSrv } from 'app/core/core';
-import { DashboardInteractions } from 'app/features/dashboard-scene/utils/interactions';
 
-import { DashboardScene } from '../dashboard-scene/scene/DashboardScene';
+import { useStarItem, useStarredItems } from './hooks';
 
-const getStarTooltips = () => ({
-  star: t('dashboard.toolbar.mark-favorite', 'Mark as favorite'),
-  unstar: t('dashboard.toolbar.unmark-favorite', 'Unmark as favorite'),
+const getStarTooltips = (title: string) => ({
+  star: t('stars.mark-as-starred', 'Mark "{{title}}" as favorite', {
+    title,
+  }),
+  unstar: t('stars.unmark-as-starred', 'Unmark "{{title}}" as favorite', {
+    title,
+  }),
 });
```

Comment:
@tomratcliffe this makes for completly unreadable tooltips. this should just say `Mark as favorite` not include the title 

<img width="668" height="223" alt="Screenshot 2025-11-08 at 17 32 20" src="https://github.com/user-attachments/assets/001e455e-4182-4818-ac58-9a65f7468af2" />